src/components/text-and-media-snippet/text-and-media-snippet.jsx

```diff
@@ -17,9 +17,17 @@ const TextAndMediaSnippet = props => (
     <div className={classNames('text-and-media-snippet', props.className)}>
         <div className="half">
             <h4>{props.title}</h4>
-            <p>
-                {props.children}
-            </p>
+            {props.children.length > 0 ?
+                <div>
+                    {/* eslint-disable */}
+                    {props.children.map((paragraph, i) => { 
```

Comment:
Just to double check, did all of the TextAndMediaSnippet's in last year's report have only one paragraph? Seems like it from a quick scan, but wanted to make sure this change doesn't impact the 2020 report.